motivation.html

```diff
@@ -277,7 +276,7 @@ <h2>Ford announces plans for a new development + research center (focussed on co
 					<div class="cd-timeline__content js-cd-content">
 						<p> Let me find details about more tech employers in Canada from UofT's Applied Computing and SFU's Big Data programs. I see <span class="ca-text">Splunk</span> and 
 							<span class="ca-text">Samsung</span>  
-						has centers in Vancouver which has teams focussed on Data Science. <span class="ca-text">Tableau</span> is in Vancouver. <span class="ca-text">Rakuten's Kobo</span> is actually from Toronto.
+						have centers in Vancouver that focusses on Data Science. <span class="ca-text">Tableau</span> is in Vancouver. <span class="ca-text">Rakuten's Kobo</span> is actually from Toronto.
```

Comment:
@srividb actually those centers have other teams as well. So we can't say it is a center focussing on DS. 
Lemme know how else we can put up.